src/commands/convert/hdf2asff.ts

```diff
@@ -23,6 +22,8 @@ export default class HDF2ASFF extends Command {
     target: flags.string({char: 't', required: true, description: 'Unique name for target to track findings across time'}),
     upload: flags.boolean({char: 'u', required: false, description: 'Upload findings to AWS Security Hub'}),
     output: flags.string({char: 'o', required: false, description: 'Output ASFF JSON Folder'}),
+    insecure: flags.boolean({char: 'I', required: false, default: false, description: 'Disable SSL verification, this is insecure.'}),
```

Comment:
imo would put that in parentheses instead of offset by a comma: "Disable SSL verification (WARNING: this is insecure)."